File: SchoolSync_SpringBoot/backend/src/main/java/com/schoolsync/dto/EnrollmentDTO.java
package com.schoolsync.dto;

import lombok.Data;
import java.time.LocalDate;

@Data
public class EnrollmentDTO {
    private Long enrollmentId;
    private Long studentId;
    private String studentName;
    private String rollNo;
    private String academicYear;
    private String className;
    private String section;
    private LocalDate enrollmentDate;
    private String status;
    private String remarks;
}

File: SchoolSync in MS.NET/Backend in .NET/DTOs/EnrollmentDto.cs
namespace SmartSchoolAPI.DTOs;

public class EnrollmentDto
{
    public int? EnrollmentId { get; set; }
    public int StudentId { get; set; }
    public string AcademicYear { get; set; } = string.Empty;
    public string Class { get; set; } = string.Empty;
    public string Section { get; set; } = string.Empty;
    public DateTime EnrollmentDate { get; set; }
    public string Status { get; set; } = string.Empty;
    public string? Remarks { get; set; }
}

